cudamapper/src/index_gpu.cuh

```diff
@@ -563,6 +577,57 @@ IndexGPU<SketchElementImpl>::IndexGPU(std::shared_ptr<DeviceAllocator> allocator
                    filtering_parameter);
 }
 
+template <typename SketchElementImpl>
+IndexGPU<SketchElementImpl>::IndexGPU(std::shared_ptr<DeviceAllocator> allocator,
+                                      const IndexCache& host_cache)
+    : first_read_id_(host_cache.first_read_id())
+    , kmer_size_(host_cache.kmer_size())
+    , window_size_(host_cache.window_size())
+    , allocator_(allocator)
+    , representations_d_(allocator)
+    , read_ids_d_(allocator)
+    , positions_in_reads_d_(allocator)
+    , directions_of_reads_d_(allocator)
+    , unique_representations_d_(allocator)
+    , first_occurrence_of_representations_d_(allocator)
+{
+    number_of_reads_                     = host_cache.number_of_reads();                     //H2D
```

Comment:
Is this really H2D? these values are on host in both cases

Reply:
that's right, fixed.